### ISSUE TITLE
Animated Dragon Wings Mod Causing Main Menu Won't Appear

### ISSUE THREAD

**Issue by umitseyhan75** [ModOrganizer2/modorganizer#683]
Just what title says. But only MO2. Didn't tried on MO1. When load the mod without MO2 virtualization there is no problem. But When I Install the mod without MO2 and open the MO2 and tick the mod and open the game in MO2 virtualization, the problem still continues.

Versions:
MO2:  2.16 (Revision e2b799b)
OS: Windows 7 Ultimate 64bit

The Last USFV: https://gist.github.com/umitseyhan75/56ce6c9911dddbb3d5dc341b17600463

The MO2 Interface: https://gist.github.com/umitseyhan75/b8f7a5553f40547b968ef2ec524642c4

**Comment by LostDragonist** [comment]
Your logs indicate that you use Comodo Internet Security (or something like that).  This has been known to cause problems with USVFS.  I don't know if there's a specific setting you can change or if you need to disable/uninstall the whole thing.

**Comment by umitseyhan75** [comment]
So I disable the antivirus program and gist the last usfv it again?

**Comment by LostDragonist** [comment]
Disabling it didn't seem to do much.  You might want to try uninstalling it.

**Comment by umitseyhan75** [comment]
OK then. If this is not gonna work, I'll try something else.
https://gist.github.com/umitseyhan75/158b85034606af2204595987841d2441

**Comment by LostDragonist** [comment]
No change with that one either.

**Comment by umitseyhan75** [comment]
https://gist.github.com/umitseyhan75/76820734582918f9cd31d59ed55d52e0
There is no AV. I don't know what else causing this if this is not too.

**Comment by LostDragonist** [comment]
That shows a change.  If that still isn't working, please set your logs to debug (settings > diagnostics > log level, set to debug), restart MO2, produce the problem, close MO2, and post the new log.

**Comment by umitseyhan75** [comment]
Tried like 5 times but gist site gives 405 Not Allowed nginx error. It's 72,9 MB. So I upload the file to my personal cloud to share with you.
Here is the link: https://yadi.sk/i/lYmbSrTMG7medg

**Comment by LostDragonist** [comment]
Hmm, okay.  I don't see any problems with your logs.  I tried the mod out and it works fine.  

You might try out a new profile/game with just this mod to see if it still causes you problems.  It could be that some other mod is conflicting with it or overwriting it.  It could be that you have too many mods, too many plugins, or you're running out of memory or something.  

I basically have no idea at this point.  Sorry.

**Comment by umitseyhan75** [comment]
I tried in new profile with onyl this mod and it fixed. But when I enabled the others the problem came back. And the other side, I mean in the default profile the it fixes only when I disable the ADW mod. So if I enabling the other mods there won't be a profile difference. I have 8 GB system and 4 GB graphic card memory. I don't think my memory running out. Number of plug-ins 165, so not even close to the limit. At the other hand MO2 shows no conflict and no overwrite. Also triden so many load order. There seem to be two solutions for now, I will either disable the ADW plugin or close all plugins except ADW.
So yeah, I basically have no idea too.
